aptos-move/aptos-vm-types/src/resolver.rs

```diff
@@ -51,9 +51,67 @@ pub trait TResourceView {
     }
 }
 
-pub trait ResourceResolver: TResourceView<Key = StateKey, Layout = MoveTypeLayout> {}
+pub trait TResourceGroupView {
+    type Key;
+    type Tag;
```

Comment:
I don't like the name Tag here it doesn't convey what it is, it probably should be resource_key and group_key